x/tokenfactory/keeper/admins_test.go

```diff
@@ -0,0 +1,86 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v7/x/tokenfactory/keeper"
+	"github.com/osmosis-labs/osmosis/v7/x/tokenfactory/types"
+)
+
+func (suite *KeeperTestSuite) TestAdminMsgs() {
```

Comment:
Lets make a follow-up issue for a plan of improved testing we should do. (How to make all of these tests table driven / more modular)

Reply:
e.g. we need more tests for attempting to mint & burn coins that are not a token factory denom.

Reply:
#1418